boards/arm/b_l072z_lrwan1/b_l072z_lrwan1_defconfig

```diff
@@ -0,0 +1,43 @@
+# Zephyr Kernel Configuration
+CONFIG_ARM=y
+CONFIG_SOC_SERIES_STM32L0X=y
+
+# Platform Configuration
+CONFIG_SOC_FAMILY_STM32=y
+CONFIG_SOC_STM32L072XZ=y
+CONFIG_BOARD_B_L072Z_LRWAN1=y
+
+# General Kernel Options
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32000000
+
+# Clock configuration
+CONFIG_CLOCK_CONTROL=y
+# SYSCLK selection
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# Use HSI source
+CONFIG_CLOCK_STM32_PLL_SRC_HSI=y
+CONFIG_CLOCK_STM32_PLL_DIVISOR=2
+# produce 32Mhz clock at PLL output
+CONFIG_CLOCK_STM32_PLL_MULTIPLIER=4
+CONFIG_CLOCK_STM32_AHB_PRESCALER=1
+CONFIG_CLOCK_STM32_APB1_PRESCALER=1
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable SPI
+CONFIG_SPI=y
```

Comment:
Remove this line. 
Default board configuration should be disabled by default (except basics and always requested: clock/gpio/console).
Others should be enabled by application